h2o-core/src/main/java/water/fvec/NewChunk.java

```diff
@@ -1537,7 +1542,7 @@ private Chunk chunkUUID() {
       if(idx >= 0)i = idx;
       else cancel_sparse(); // for now don't bother setting the sparse value
     }
-    _is[i] = _sslen;
+    if (_is != null) _is[i] = _sslen;
```

Comment:
_ss and _is seems to be related to each other, documentation says: "_ss holds appended string bytes (with trailing 0), _is[] holds offsets into _ss[]"

You can see code like this: "else if (_ss != null) int sidx = _is[i];" in NewChunk.

That makes me wonder if there is not a deeper issue with _is and _ss. That is why I am questioning this fix.

Perhaps @tomasnykodym can explain the relation of _is and _ss.